docs/app/Examples/views/Statistic/Types/StatisticGroupExample.js

```diff
@@ -0,0 +1,24 @@
+import React, {Component} from 'react';
+import {Statistic} from 'stardust';
+const {Statistics, Label, Value} = Statistic;
```

Comment:
This would only affect the group sub-component. I think for now then we'll keep parity with the class names, sd-statistic-statistics inside <Statistic.Group /> and also, the Semantic UI doc site. If we feel otherwise later we can refactor.